spec.md

```diff
@@ -814,7 +815,7 @@ The request provides these query string parameters as useful hints for brokers.
 ##### cURL
 ```
 $ curl 'http://username:password@broker-url/v2/service_instances/:instance_id?accepts_incomplete=true
-  &service_id=service-id-here&plan_id=plan-id-here' -X DELETE -H "X-Broker-API-Version: 2.11"
+  &service_id=service-id-here&plan_id=plan-id-here' -X DELETE -H "X-Broker-API-Version: 2.12"
```

Comment:
Oh, my bad, I see that those sections omitted the header.  Yeah, I'll fix that in a later PR.